docs/resources/dashboard.md

```diff
@@ -7573,6 +7574,14 @@ Optional:
 
 
 
+<a id="nestedblock--widget--group_definition--widget--sunburst_definition--request--style"></a>
+### Nested Schema for `widget.group_definition.widget.sunburst_definition.request.style`
+
+Optional:
+
+- `palette` (String) A color palette to apply to the widget. The available options are available at: https://docs.datadoghq.com/dashboards/widgets/timeseries/#appearance.
```

Comment:
**question:** Is it intentional that we're linking to the timeseries docs here? If these docs are generated then we may need to fix it upstream